integration-junit4/pom.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.github.dockerunit</groupId>
+        <artifactId>parent</artifactId>
+        <version>${revision}</version>
+    </parent>
+
+    <artifactId>integration-junit4</artifactId>
+
+    <description>Dockerunit JUnit 4 Integration Tests Module</description>
+
+    <properties>
+        <!-- Config -->
+        <build.config.dir>${project.basedir}/../build</build.config.dir>
+        <junit4.version>4.13.1</junit4.version>
+        <rest-assured.version>4.2.0</rest-assured.version>
+        <springframework.version>5.3.3</springframework.version>
```

Comment:
The initial concept I was going for was to define all versions that we'd like to ensure are the same across modules (if ever used) in the parent pom `dependencyManagement` section. And only define the versions in the modules if they need to be a particular version for the module in question.

Now that there are two modules depending on `junit4` maybe we should just move `junit4.version` to the parent pom and define under its `dependencyManagement` the `junit4` dependency. Then both here and in `junit4` we only need to declare the `junit4` dependency.

Same would apply to `rest-assured` and `springframework`

Reply:
alright